knowledge-base/snyk/action/action-security.yml

```diff
@@ -0,0 +1,5 @@
+name: 'Snyk-dokker action'
```

Comment:
@Devils-Knight github.com/snyk/action returns a 404 page not found. Can you please change the file name to `snyk/actions`?